tests/software/last/lastdb/test.yml

```diff
@@ -0,0 +1,42 @@
+- name: last lastdb test_last_lastdb
+  command: nextflow run tests/software/last/lastdb -entry test_last_lastdb -c tests/config/nextflow.config
+  tags:
+    - last
+    - last/lastdb
+  files:
+    - path: output/last/lastdb/test.bck
+      md5sum: 5519879b9b6c4d1fc508da7f17f88f2e
+    - path: output/last/lastdb/test.des
+      md5sum: 3a9ea6d336e113a74d7fdca5e7b623fc
+    - path: output/last/lastdb/test.prj
+      md5sum: 489715f14b0fea6273822696e72357f9
+    - path: output/last/lastdb/test.sds
+      md5sum: 2cd381f4f8a9c52cfcd323a2863eccb2
+    - path: output/last/lastdb/test.ssp
+      md5sum: 4137fb6fe9df2b3d78d5b960390aac7b
+    - path: output/last/lastdb/test.suf
+      md5sum: 1895efa8653e8e9bd3605cff0408ed33
+    - path: output/last/lastdb/test.tis
+      md5sum: b7c40f06b1309dc6f37849eeb86dfd22
+
+- name: last lastdb test_last_lastdb_gzipped_input
+  command: nextflow run tests/software/last/lastdb -entry test_last_lastdb_gzipped_input -c tests/config/nextflow.config
+  tags:
+    - last/lastdb
+    - last_lastdb_gzipped_input
+    - last
```

Comment:
```suggestion
    - last
    - last/lastdb
```

Reply:
Sorry, can't push commits to your fork @charles-plessy but would you mind making this change too? I think that is why the linting is failing.